test/test_build.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+def open_test_json_file(name)
+  File.new(File.join(File.dirname(__FILE__), 'json', name))
+end
+
+# Build entity tests
+class TestBuild < MiniTest::Test
+  def test_get_builds
+    stub_request(:get, %r{/builds})
+      .to_return(body: open_test_json_file('builds_list.json'),
+                 status: 200)
+
+    client = OpenshiftClient::Client.new 'https://localhost:8080/oapi'
```

Comment:
iirc, there's no need to specify /oapi when initializing the client - this is the default anyway. (it does work, but is not necessary)